templates/default/password_toggle.js

```diff
@@ -0,0 +1,52 @@
+$(document).ready(function() {
+	let password_input = document.getElementById('curl_password');
```

Comment:
the modification also means that we no longer have a direct bundling to exactly this element but the functionality could also be used for other password inputs.